app/Http/Requests/RegisterRequest.php

```diff
@@ -22,7 +22,9 @@ public function authorize(): bool
     public function rules(): array
     {
         return [
-        
+            'email' => 'required|email',
+            'password' => 'required|min:6|max:100',
+            'confirm_password' => 'required|same:password'
```

Comment:
instead of checking 'confirm_password' separately. Investigate and add the 'confirmed' rule to the 'password' .
https://laravel.com/docs/10.x/validation#rule-confirmed